spec/super_good/solidus_taxjar/reporting_spec.rb

```diff
@@ -1,13 +1,60 @@
 require "spec_helper"
 
 RSpec.describe SuperGood::SolidusTaxjar::Reporting do
-  describe "#report_transaction" do
-    subject { described_class.new(api: dummy_api).report_transaction(order) }
+  let(:dummy_api) { instance_double ::SuperGood::SolidusTaxjar::Api }
+  let(:order) { build :order, completed_at: 1.days.ago }
+  let(:reporting) { described_class.new(api: dummy_api) }
+
+  describe "#refund_and_create_transaction" do
+    subject { reporting.refund_and_create_new_transaction(order) }
+
+    it "refunds the transaction and creates a new one in TaxJar" do
```

Comment:
Out of curiosity, why aren't we using VCR for this test but we are for the others?

Reply:
This was the only test case where we explicitly cared about an error class from an external library. (A `Taxjar::Error` subclass.)

I would like to use VCR more but this is the only case where it seemed absolutely necessary to help our future selves make sense of the extension code being the way it is: `@api.create_transaction(order)` will not get executed if `@api.refund_transaction(order)` fails beforehand.